modules/enabling-intel-gaudi-ai-accelerators.adoc

```diff
@@ -18,44 +18,22 @@ endif::[]
-. From the {productname-short} dashboard, click *Settings* -> *Accelerator profiles*.
-+
-The *Accelerator profiles* page appears, displaying existing accelerator profiles. To enable or disable an existing accelerator profile, on the row containing the relevant accelerator profile, click the toggle in the *Enable* column.
-. Click *Create accelerator profile*. 
-+
-The *Create accelerator profile* dialog opens.
-. In the *Name* field, enter a name for the Intel Gaudi AI Accelerator.
-. In the *Identifier* field, enter a unique string that identifies the Intel Gaudi AI Accelerator, for example, `habana.ai/gaudi`. 
-. Optional: In the *Description* field, enter a description for the Intel Gaudi AI Accelerator.
-. To enable or disable the accelerator profile for the Intel Gaudi AI Accelerator immediately after creation, click the toggle in the *Enable* column.
-. Optional: Add a toleration to schedule pods with matching taints.
-.. Click *Add toleration*. 
-+
-The *Add toleration* dialog opens.
-.. From the *Operator* list, select one of the following options:
-* *Equal* - The *key/value/effect* parameters must match. This is the default.
-* *Exists* - The *key/effect* parameters must match. You must leave a blank value parameter, which matches any.
-.. From the *Effect* list, select one of the following options:
-* *None* 
-* *NoSchedule* - New pods that do not match the taint are not scheduled onto that node. Existing pods on the node remain.
-* *PreferNoSchedule* - New pods that do not match the taint might be scheduled onto that node, but the scheduler tries not to. Existing pods on the node remain.
-* *NoExecute* - New pods that do not match the taint cannot be scheduled onto that node. Existing pods on the node that do not have a matching toleration are removed.
-.. In the *Key* field, enter the toleration key `habana.ai/gaudi`. The key is any string, up to 253 characters. The key must begin with a letter or number, and may contain letters, numbers, hyphens, dots, and underscores.
-.. In the *Value* field, enter a toleration value. The value is any string, up to 63 characters. The value must begin with a letter or number, and may contain letters, numbers, hyphens, dots, and underscores.
-.. In the *Toleration Seconds* section, select one of the following options to specify how long a pod stays bound to a node that has a node condition. 
-** *Forever* - Pods stays permanently bound to a node. 
-** *Custom value* - Enter a value, in seconds, to define how long pods stay bound to a node that has a node condition.
-.. Click *Add*.
-. Click *Create accelerator profile*.
 
 .Verification
-* From the *Administrator* perspective, the following Operators appear on the *Operators* -> *Installed Operators* page.
+* From the *Administrator* perspective, the following operators appear on the *Operators* -> *Installed Operators* page.
```

Comment:
I think it was correct in the first place. We always capitalize Operators.

Reply:
Updated, thanks!